src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -48,7 +48,7 @@
       <template v-for="(orgPerson, index) in currentPeopleAndRoles">
         <v-row
           v-if="!(wasReplaced(orgPerson) && wasRemoved(orgPerson))"
-          class="people-roles-content section-container"
+          class="people-roles-content py-3"
```

Comment:
After:

![image](https://user-images.githubusercontent.com/10002889/232609982-509305c6-c7ed-4325-87b1-03d4794c9980.png)

After for a different filing type with several directors:

![image](https://user-images.githubusercontent.com/10002889/232610096-0ee73bb1-250e-49f8-8774-7229e9fde000.png)

Reply:
I'm reverting this change and creating a new bug ticket for this because it looks different locally and in Dev.